src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/UpgradeToGeneratedRegexAnalyzerTests.cs

```diff
@@ -182,6 +182,58 @@ public static void Main(string[] args)
             }.RunAsync();
         }
 
+        [Fact]
+        public async Task CodeFixSupportsInvalidPatternFromWhichOptionsCanBeParsed()
+        {
+            string pattern = ".{99999999999}"; // throws during real parse
```

Comment:
It would be better to not flag / offer a fix in this case too, but apparently it was not caught during the parse in validation